ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/RedHat.yml

```diff
@@ -92,17 +92,6 @@ Additional_Build_Tools_RHEL7_s390x:
   - libstdc++.s390                # a dependency required for executing a 32-bit C binary
   - yum-utils                     # yumdownloader required for devkit creation
 
-Java_RHEL8:
```

Comment:
Since we still have the reference to `Java_RHEL8` above this shouldn't be removed